docs/api/meta.md

```diff
@@ -0,0 +1,165 @@
+# About
+
+This is the documentation of the API for accessing the API Metadata (`meta`) for `fluview`, `twitter`, `wiki`,
+and `delphi` data sources of the [Delphi](https://delphi.cmu.edu/)'s epidemiological data.
+
+## API Metadata
+
+... <!-- TODO -->
+
+## Contributing
+
+If you are interested in contributing:
+
+- For development of the API itself, see the
+  [development guide](docs/epidata_development.md).
+- To suggest changes, additions, or other ways to improve,
+  [open an issue](https://github.com/cmu-delphi/delphi-epidata/issues/new)
+  describing your idea.
+
+## Citing
+
+We hope that this API is useful to others outside of our group, especially for
+epidemiological and other scientific research. If you use this API and would
+like to cite it, we would gratefully recommend the following copy:
+
+> David C. Farrow,
+> Logan C. Brooks,
+> Aaron Rumack,
+> Ryan J. Tibshirani,
+> Roni Rosenfeld
+> (2015).
+> _Delphi Epidata API_.
+> https://github.com/cmu-delphi/delphi-epidata
+
+## Data licensing
+
+The data surfaced through this API is just a carefully curated mirror of data acquired from .... <!-- TODO -->
```

Comment:
this one is actually delphi's own original source

Reply:
changed to the delphi licensing wording.